api/test/mock-data/course-mock-data.js

```diff
@@ -0,0 +1,57 @@
+module.exports.POST_SIMPLE_COURSE = {
+    approved: true,
+    details: { 
+        name: "Learning 101",
+        description: "hii",
+        translated_language: "English",
+        location: "Michigan",
+        iso: "153",
+    }
+}
+
+module.exports.POST_SIMPLE_COURSE_EXPECTED = this.POST_SIMPLE_COURSE;
+
+module.exports.POST_MISSING_FIELD_COURSE = {
```

Comment:
One quick change to the `CourseDetail` schema: Can you make all of the `CourseDetails` fields required? 

Also, keep the `translated_langauge` default value as "English," don't give `name` a default value, and the rest of the fields a default value of "". The name shouldn't have a default value since every course should have a name.

Reply:
Update: can you add two more fields, `adminName` and `email`, which should both be required and have no default value.

Reply:
const CourseDetails = new mongoose.Schema({
  adminName: { type: String, required: true },
  adminEmail: { type: String, required: true },
  name: { type: String, required: true },
  alternativeName: { type: String, required: true, default: '' },
  description: { type: String, required: true, default: '' },
  iso: { type: String, required: true, default: '' },
  glotto: { type: String, required: true, default: '' },
  translated_language: { type: String, required: true, default: 'English' },
  population: { type: String, required: true, default: '' },
  location: { type: String, required: true, default: '' },
  link: { type: String, required: true, default: '' },
});